index.html

```diff
@@ -140,20 +140,22 @@ <h2>カフェで読書＆コワーキング。快適のフロアスペース空
           </p>
         </div>
       </section>
-      <section id="description" class="container-xl">
-        <div class="image row row-cols-2 row-cols-md-4 no-gutters">
-          <img src="./images/description-01.jpg" class="col" />
-          <img src="./images/description-02.jpg" class="col" />
-          <img src="./images/description-03.jpg" class="col" />
-          <img src="./images/description-04.jpg" class="col" />
-        </div>
-        <div class="inner text-center bg-light">
-          <h2 class="mincho">人と人、人と本が繋がるまったく新しい快適時間。</h2>
-          <p class="m-0">
-            月額プラン以外にビジター料金でのご利用も可能です。通いやすい時間や曜日に合わせてプランをお選びください。<br />
-            ご登録プランは後の変更も可能ですので、資格試験等の目標がある場合など環境の変化に合わせて柔軟に対応しております。WiFiも電源も標準設備しており、<br />
-            時間を気にせず仕事や勉強に取り組めます。カフェでの長居は気が引けるという方にも心置きなくご利用いただけます。
-          </p>
+      <section id="description">
+        <div class="block container-fluid p-0">
```

Comment:
余白は必要なのでcontainerで良いと思います

Reply:
余白は`#description`の`padding`で指定していますが、まだ足りないところがあるのでしょうか？
```suggestion
        <div class="block container p-0">
```
としてしまうと、画面の幅がbootstrapのブレークポイントより少し小さいとき（597 ~ 767px）に余計な余白ができてしまいます。

Reply:
私は、container-fluidにcssでpaddingでつけるならcontainerにしてcssのpaddingは消して良いのかな〜と思ったんですが、takahiroさんのおっしゃることも分かるのでayakoさんに聞いてみます

Reply:
今のままでマージして大丈夫だそうです！！
ayakoさん曰く、container-fluid container-{break-point} 
https://getbootstrap.jp/docs/4.4/layout/overview/
みたいな感じで細かく指定する方法もあるそうです
でもcssの方がpadding細かくカスタマイズできるので画面幅が広い時でも今のpaddingが良いならこのままでいきましょう〜すみません！

Reply:
そうですね、今回はcontainer-xlの固定幅をカスタマイズしたかったので、それならばcontainer-fluidにしてmax-widthを指定すれば良いかなと思いこのようにしました。